openshift-knative-operator/pkg/monitoring/monitoring.go

```diff
@@ -109,3 +121,74 @@ func InjectNamespaceWithSubject(resourceNamespace string, subjectNamespace strin
 		return nil
 	}
 }
+
+func LoadServingMonitoringPlatformManifests() ([]mf.Manifest, error) {
+	rbacPath := os.Getenv(servingSMRbacManifestPath)
+	if rbacPath == "" {
+		return nil, fmt.Errorf("failed to get the Serving sm rbac manifest path")
+	}
+	rbacManifest, err := mf.NewManifest(rbacPath)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
similar to https://github.com/openshift-knative/serverless-operator/pull/841/files#diff-9441c1af1419ac92923cabd71411c98c7702acbdad9baec26fddf7a8a4bb221bR134-R141